app/views/browse/route.html

```diff
@@ -56,15 +56,15 @@
                   <dt>Hostname<span ng-if="route.status.ingress.length > 1">s</span>:</dt>
                   <dd>
                     <span ng-if="!route.status.ingress">
-                      {{route | routeLabel}}
+                      {{route | routeLabel : null : true}}
                       <span data-toggle="popover" data-trigger="hover" data-content="The route is not accepting traffic yet because it has not been admitted by a router." style="cursor: help; padding-left: 5px;">
                         <status-icon status="'Pending'" disable-animation></status-icon>
                         <span class="sr-only">Pending</span>                        
                       </span>
                     </span>  
                     <div ng-repeat="ingress in route.status.ingress">
                       <span ng-if="(route | isWebRoute)">
-                        <a href="{{route | routeWebURL : ingress.host}}" target="_blank">{{route | routeLabel : ingress.host}}</a>
+                        <a href="{{route | routeWebURL : ingress.host}}" target="_blank">{{route | routeLabel : ingress.host : true}}</a>
```

Comment:
sorry this isn't new but i just noticed it, this should be using ng-href and not href, mind fixing that here?